pallets/loan/src/math.rs

```diff
@@ -267,3 +298,62 @@ mod tests {
 		)
 	}
 }
+
+#[test]
+fn valid_write_off_groups() {
```

Comment:
It also be interesting to test for invalid write off groups

Reply:
it is already tested down below